File: Java/2194_Cells_in_a_Range_on_an_Excel_Sheet.java
class Solution {
    public List<String> cellsInRange(String s) {
        char startX = s.charAt(0);
        char startY = s.charAt(1);
        char endX = s.charAt(3);
        char endY = s.charAt(4);
        
        List<String> result = new ArrayList<>();
        char X = startX;
        char Y = startY;
        while(X <= endX){
            while(Y <= endY){
                String item = (X + "")+(Y+"");
                result.add(item);
                Y++;
            }
            X++;
            Y = startY;
        }
        return result;
    }
}

File: Python3/2194_Cells_in_a_Range_on_an_Excel_Sheet.py
class Solution:
    def cellsInRange(self, s: str) -> List[str]:
        result = []
        sc, ec = s[0], s[3]
        si, ei = int(s[1]), int(s[4])
        i = si
        while(ord(sc) <= ord(ec)):
            while(i <= ei):
                result.append(sc + str(i))
                i += 1
            i = si
            sc = chr(ord(sc) + 1)
            
        return result
